File: Python/Vehicle.py
#deklarasi class
class Vehicle:

    #private atribut
    __platNomor = ""
    __merk = ""
    __tahunProduksi = ""
    __warna = ""

    #constructor
    def __init__(self, platNomor = "", merk = "", tahunProduksi = "", warna = ""):
        self.__platNomor = platNomor
        self.__merk = merk
        self.__tahunProduksi = tahunProduksi
        self.__warna = warna

    #setter
    def setplatNomor(self, platNomor):
        self.__platNomor = platNomor
    def setmerk(self, merk):
        self.__merk = merk
    def settahunProduksi(self, tahunProduksi):
        self.__tahunProduksi = tahunProduksi
    def setwarna(self, warna):
        self.__warna = warna
    
    #getter
    def getplatNomor(self):
        return self.__platNomor
    def getmerk(self):
        return self.__merk
    def gettahunProduksi(self):
        return self.__tahunProduksi
    def getwarna(self):
        return self.__warna
    

File: C++/Vehicle.cpp
#pragma once
// library
#include <iostream>
#include <string>

using namespace std;

class Vehicle
{
    // atribut private
private:
    string plat;
    string merk;
    string tahun;
    string warna;

public:
    // constructor
    Vehicle()
    {
    }
    Vehicle(string plat, string merk, string tahun, string warna)
    {
        this->plat = plat;
        this->merk = merk;
        this->tahun = tahun;
        this->warna = warna;
    }

    // setter
    void setPlat(string plat)
    {
        this->plat = plat;
    }
    void setMerk(string merk)
    {
        this->merk = merk;
    }
    void setTahun(string tahun)
    {
        this->tahun = tahun;
    }
    void setWarna(string warna)
    {
        this->warna = warna;
    }

    // getter
    string getPlat()
    {
        return this->plat;
    }
    string getMerk()
    {
        return this->merk;
    }
    string getTahun()
    {
        return this->tahun;
    }
    string getWarna()
    {
        return this->warna;
    }

    // destructor
    ~Vehicle()
    {
    }
};